packages/cli/src/lib/index.js

```diff
@@ -217,6 +220,15 @@ export const getPaths = () => {
   }
 }
 
+export const getConfig = () => {
+  try {
+    return getRedwoodConfig()
+  } catch (e) {
+    console.error(c.error(e.message))
+    process.exit(0)
```

Comment:
A 0 exit code means success
```suggestion
    process.exit(1)
```

Reply:
I just copied the code from getPaths, didn't read through it properly :) I'll change that also